ftl/ftl_node_benchmark_yaml.py

```diff
@@ -71,5 +103,23 @@ def benchmark_step(iterations, app_dir):
     ]
 
 
+def app_size_test_step(app_dir, iterations, gen_files):
+    name = 'gcr.io/ftl-node-test/benchmark_%s:latest' % app_dir
+    return [
+        # First build the image
+        {
+            'name':
+            'bazel/ftl/node/benchmark:node_benchmark_image',
+            'args': [
+                '--base', _NODE_BASE, '--name', name, '--directory',
+                os.path.join(_DATA_DIR + app_dir), '--description', app_dir,
+                '--iterations',
+                str(iterations), '--gen_files',
```

Comment:
Do you need str() here?

Reply:
it is in the original dep_size_tests. I have changed both